### PR TITLE
Switch schema `version` type to `str`

### PR DESCRIPTION
**Pull request by afshin**
In JupyterLab, the `version` field is defined as a `string` and in `jupyter_events` it is defined as a number.

The least disruptive resolution to this is to update `jupyter_events` to switch its schema `version` to be a `str`.

### CODE REVIEW

**Comment by Carreau** [comment]
Rebased on master and re-enabled workflows to run tests.

File: docs/user_guide/event-schemas.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ print(logger.schemas)
 Validator class: Draft7Validator
 Schema: {
   "$id": "myapplication.org/example-event",
-  "version": 1,
+  ""version": 1",
```
Comment (afshin) [inline docs/user_guide/event-schemas.md:41]:
```suggestion
  "version": "1",
```

Comment (Carreau) [reply docs/user_guide/event-schemas.md:41]: Rah... I'll make a PR

Comment (afshin) [reply docs/user_guide/event-schemas.md:41]: 🙏

**Pull request merged**